### DISCUSSION TITLE
What's the right maximum line length?

### DISCUSSION THREAD

**Discussion by mbolivar-nordic** [zephyrproject-rtos/zephyr#28327] [Q&A] [answered]
What is the correct maximum number of characters in a line of C, and why?

**Comment by nashif** [comment, marked as answer]
There is no correct answer...
If we follow the Linux style rules, i.e. we follow them as they evolve, then it should be 100 now.. But we touched on this in the TSC and established we should consider taking a specific version of the Linux guidelines as a baseline with some exceptions and additions and increase that baseline version and adapt exceptions in the future, or we should just fork the guidelines and have our own and decouple from Linux. Whatever option we take, we (Zephyr) should decide if it is going 80 or 100, not Linux :)

**Comment by mbolivar-nordic** [comment]
THIS THING I'M WRITING NOW IS A SUBSTITUTE FOR A NOISY COMMENT ABOUT A STRONGLY HELD PERSONAL OPINION STATED AS FACT

PROMINENTLY FEATURING AD HOMINEM ATTACKS AND SPELLING ERRRZ

(curious to see how this compares in the UI to the comment marked "as the answer")
